src/Microsoft.ML.AutoML/API/AutoCatalog.cs

```diff
@@ -587,6 +587,44 @@ internal SweepableEstimator[] CatalogFeaturizer(string[] outputColumnNames, stri
             return new SweepableEstimator[] { SweepableEstimatorFactory.CreateOneHotEncoding(option), SweepableEstimatorFactory.CreateOneHotHashEncoding(option) };
         }
 
+        internal MultiModelPipeline ImagePathFeaturizer(string outputColumnName, string inputColumnName)
```

Comment:
Are you planning on adding support for having a folder and not just an image column?

Reply:
Not for now, this only support full image path